src/routes/messages.js

```diff
@@ -16,10 +16,11 @@ module.exports = (() => {
     *
 	* @apiHeader {String} token Authorization Bearer Token.
     * @apiParam {String} mid Message's id.
+	* @apiParam (Request body) {String} emoji Emoji reaction.
```

Comment:
I think this is expected in the body, right? I use the parens just to make it a separate grouping so it shows up under a separate section called Request body in the docs for this endpoint.